packages/block-library/src/video/video-player.native.scss

```diff
@@ -13,7 +13,7 @@ $play-icon-size: 50;
 	align-items: center;
 	align-self: center;
 	position: absolute;
-	background-color: #e9eff3;
+	background-color: transparent;
```

Comment:
not directly related with this PR but updated this intentionally, there was a whitish color with 0.3 opacity on the videos which made videos look like they lost saturation.